mlflow/terraform/gcp/main.tf

```diff
@@ -4,6 +4,7 @@ resource "kubernetes_namespace" "mlflow" {
     labels = {
       "app.kubernetes.io/managed-by" = "plural"
       "app.plural.sh/name" = "mlflow"
+      "platform.plural.sh/sync-target" = "pg"
     }
   }
```

Comment:
looks like gcp/azure aren't having their object stores provisioned?

Reply:
We also don't have recipes for GCP and Azure yet. Given the past experience with MLFlow now we should test those cloud providers rigorously before releasing those recipes.